dll/cpl/desk/background.c

```diff
@@ -652,6 +653,7 @@ OnBrowseButton(HWND hwndDlg, PBACKGROUND_DATA pData)
 
     hwndBackgroundList = GetDlgItem(hwndDlg, IDC_BACKGROUND_LIST);
     himl = ListView_GetImageList(hwndBackgroundList, LVSIL_SMALL);
```

Comment:
what if himl is NULL ?

Reply:
Not sure how I would error check this, I've found other ROS code that doesn't take this into account.